content/influxdb/cloud/notebooks/manage-notebooks.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Manage notebooks
+description:
+weight: 102
+influxdb/v2.0/tags:
+menu:
+  influxdb_2_0:
+    name: Manage notebooks
+    parent: Notebooks
+---
+{{% note %}}
+**Notebooks is currently an early access feature.**
+[Submit a request](https://w2.influxdata.com/notebooks-early-access/ ) to be added to the queue, and we will send you a confirmation when you’ve been added to early access.
+{{% /note %}}
+
+View, update, and delete notebooks in the UI.
+
+## View notebooks
```

Comment:
Thoughts on combining "View or update a notebook" being first only diff is last step?

 3. To update, edit the notebook's cells and content. Changes are saved automatically.